### ISSUE TITLE
Something wrong in training VRD

### ISSUE THREAD

**Issue by handsomefengfeng** [StanfordVL/ReferringRelationships#14]
Set the configure '--save-best-only' and find that the val_loss just didn't improve from the first epoch.

Through the tensorboard on log, the performance on training set was normal, in which the loss trend to descend. However, the performance on val set was unexpected, in which the val_loss didn't descend including all metric on val not convergence.

**Comment by handsomefengfeng** [comment]
> Set the configure '--save-best-only' and find that the val_loss just didn't improve from the first epoch.
> 
> Through the tensorboard on log, the performance on training set was normal, in which the loss trend to descend. However, the performance on val set was unexpected, in which the val_loss didn't descend including all metric on val not convergence.
